src/main/helm/helm-release-manager.ts

```diff
@@ -107,25 +128,41 @@ export async function getValues(name: string, { namespace, all = false, kubeconf
     "--kubeconfig", kubeconfigPath,
   );
 
-  return execHelm(args);
+  const result = await execHelm(args);
+
+  if (result.callWasSuccessful) {
+    return result.response;
+  }
+
+  throw result.error;
 }
 
-export async function getHistory(name: string, namespace: string, kubeconfigPath: string) {
-  return json.parse(await execHelm([
+export async function getHistory(name: string, namespace: string, kubeconfigPath: string): Promise<JsonValue> {
+  const result = await execHelm([
     "history",
     name,
     "--output", "json",
     "--namespace", namespace,
     "--kubeconfig", kubeconfigPath,
-  ]));
+  ]);
+
+  if (result.callWasSuccessful) {
+    return json.parse(result.response);
+  }
+
+  throw result.error;
 }
 
-export async function rollback(name: string, namespace: string, revision: number, kubeconfigPath: string) {
-  await execHelm([
+export async function rollback(name: string, namespace: string, revision: number, kubeconfigPath: string): Promise<void> {
+  const result = await execHelm([
     "rollback",
     name,
     `${revision}`,
     "--namespace", namespace,
     "--kubeconfig", kubeconfigPath,
   ]);
+
+  if (!result.callWasSuccessful) {
```

Comment:
Note: there's something brewing elsewhere to remove duplication from error handling, and formalise it. Let's look at that later.